### PR TITLE
feat(cli): add entities workspace [KHCP-7981]

### PR DESCRIPTION
**Pull request by adamdehaven**
# Summary

Add the new `entities` workspace in preparation for moving the `shared-ui-components/packages/entities/*` packages to this repository.

## PR Checklist

* [ ] **Naming & Structure:** the files and package structure use the conventions outlined in the [Creating a Package docs](https://github.com/Kong/public-ui-components/blob/main/docs/creating-a-package.md).
* [ ] **Tests pass:** check the output of all package unit and/or component tests.
  * [ ] If this PR is the result of a bug, test coverage was added accordingly.
* [ ] **Functional:** all changes do not break existing APIs, but if so, a BREAKING CHANGE commit is in place to bump the major version.
* [ ] **Conventional Commits** all commits follow the conventional commit standards [outlined in the main README](https://github.com/Kong/public-ui-components#committing-changes).
* [ ] **Docs:** includes a technically accurate README, and the docs have been updated accordingly based on the changes in this PR.

### CODE REVIEW

File: packages/core/cli/src/__template__/sandbox/tsconfig.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "baseUrl": "."
+    "baseUrl": "."{%%SANDBOX_TSCONFIG_PATHS%%
+    }
```
Comment (sumimakito) [inline packages/core/cli/src/__template__/sandbox/tsconfig.json:5]:
```suggestion
    "baseUrl": "."{%%SANDBOX_TSCONFIG_PATHS%%}
```

The placeholder is broken due to the whitespaces.

Comment (adamdehaven) [reply packages/core/cli/src/__template__/sandbox/tsconfig.json:5]: This is just the template file; formatting will be wrong here it's just used to generate the actual file

Comment (adamdehaven) [reply packages/core/cli/src/__template__/sandbox/tsconfig.json:5]: Good catch, thanks!

**Review by Leopoldthecoder — APPROVED** [review]
LGTM🚀

**Review by sumimakito — APPROVED** [review]
LGTM, thanks!

**Pull request merged**